### ISSUE TITLE
Support single quotes in html templates

### ISSUE THREAD

**Issue by ixevix** [mvdschee/web-accessibility#37]
**Is your feature request related to a problem? Please describe.**
We are using single quotes in our ejs template to work around it breaking when we serialize it into JSON. The Web Accessibility extension still complains about for example the alt text missing though it's there, it's just using single quotes.

**Describe the solution you'd like**
Support single quotes or allow it to be toggled in options.

**Comment by ixevix** [comment]
Actually looking at the server code seems that the regexp that alt is using has hardcoded the double quotes. Other tags don't seem to do this so this might just be a bug.

**Comment by mvdschee** [comment]
He @ixevix will look in to how I can fix this, might create a lot of duplicated code so will check how I can get around that issue.

**Comment by alevann** [comment]
Any news on this issue? We use JavaScript Standard Style which enforces single quotes, this conflict causes all img tags in jsx files to always show an error no matter what.
